set-up-location-driven-geotriggers/README.md

```diff
@@ -0,0 +1,46 @@
+# Set up location driven geotriggers
+
+Create a notification every time a given location data source has entered and/or exited a set of features or graphics.
+
+![Geotriggers](set-up-location-driven-geotriggers.png)
+
+## Use case
+
+Geotriggers can be used to notify users when they have entered or exited a geofence by monitoring a given set of features or graphics. They could be used to display contextual information to museum visitors about nearby exhibits, notify hikers when they have wandered off their desired trail, notify dispatchers when service workers arrive at a scene, or more.
+
+## How to use the sample
+
+Observe a virtual walking tour of the Santa Barbara Botanic Garden. Information about the user's current Garden Section, as well as information about nearby points of interest within 10 meters will display or be removed from the UI when the user enters or exits the buffer of each feature.
+
+## How it works
+
+1. Create a `GeotriggerFeed` with an `AbstractLocationDataSource` class (in this case, a `SimulatedLocationDataSource`).
```

Comment:
We don't have `AbstractLocationDataSource`
```suggestion
1. Create a `GeotriggerFeed` with a `LocationDataSource` class (in this case, a `SimulatedLocationDataSource`).
```

Reply:
I think I'm missing to refactor the readme to match the API spec. Will add this to my to-do before the PR.